numba/cpython/unicode.py

```diff
@@ -842,6 +830,56 @@ def _adjust_indices(length, start, end):
     return start, end
 
 
+@overload_method(types.UnicodeType, 'startswith')
+def unicode_startswith(s, prefix, start=None, end=None):
+    if not (start is None or isinstance(start, (types.Omitted,
```

Comment:
Should this use `numba.core.cgutils.is_nonelike` oppose to both checking for `None`, `types.NoneType` and `types.Omitted`?

Reply:
Revised using this func.